### ISSUE TITLE
Exiting Zen mode?

### ISSUE THREAD

**Issue by kendaleiv** [onlywei/explain-git-with-d3#26]
Is there a way to exit Zen mode (without refreshing the browser)? If so, I haven't been able to figure it out.

**Comment by JakeGinnivan** [comment]
Nope, I never got around to putting that in. Did you want to submit a pull request?
